src/main/java/uk/ac/ebi/eva/pipeline/runner/exceptions/UnexpectedErrorReadingFileException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.runner.exceptions;
+
+import java.io.IOException;
+
+/**
+ * Unexpected error while reading or processing a file
+ */
+public class UnexpectedErrorReadingFileException extends Exception {
+
+    public UnexpectedErrorReadingFileException(String filePath, IOException e) {
+        super("Unexpected error reading file '" + filePath + "'", e);
```

Comment:
so this is the only exception that calls its parent with the nested exception? is this intended? why?